docs/content/Theming/06_browser_color.md

```diff
@@ -43,5 +43,5 @@ Below are usage examples for both the Angular configuration phase, and during ru
     })
 </hljs>
 
-For api reference please visit [$mdThemingProvider](/api/service/$mdThemingProvider#enableBrowserColor) documentation.
+For api reference please visit <a ng-href="/api/service/$mdThemingProvider#enableBrowserColor">$mdThemingProvider</a> documentation.
```

Comment:
As with the other link, using markdown syntax returns the wrong URL. 

This incorrect behavior was fixed previously in areas of the code, but it seems these two links fell through the cracks.

See https://github.com/angular/material/commit/0d7fbadc4a4c230d7b6af6bffeeef84fad655edd